.gitignore

```diff
@@ -38,5 +38,8 @@ cmake-build-*
 # vscode settings
 source/.vs*
 
+# build
+source/build
```

Comment:
why is this back here? There should not be any build folder. Is it because you have one in vscode?

Reply:
Yes I have a build folder. You don't?

Reply:
nope the toolchain in clion does not put it there. This is a small detail keep it there then